src/syscall_mock/mod.rs

```diff
@@ -36,7 +35,10 @@ impl SyscallMock {
             context: context.clone(),
             tracee_pid,
             protocol,
-            unmocked_commands: unmocked_commands.to_vec(),
+            unmocked_commands: unmocked_commands
+                .iter()
+                .map(|c| Command::executable_full_path(c))
+                .collect(),
```

Comment:
I'm not sure it's a good idea to change the behaviour of `unmockedCommands` in this PR. Just to keep things small. Also, is this change tested?

Reply:
Ok, I changed it back, and implemented `compare_executables` to achieve the same result which keeps path resolution logic in `protocol::command`. But the new implementation does perform more `$PATH` lookups.